src/erlfmt_recomment.erl

```diff
@@ -217,13 +217,19 @@ insert_nested({'case', Meta, Expr0, Clauses0}, Comments0) ->
     Clauses = insert_expr_container(Clauses0, Comments1),
     {{'case', Meta, Expr, Clauses}, []};
 insert_nested({'receive', Meta, Clauses0}, Comments0) ->
-    Clauses = insert_expr_container(Clauses0, Comments0),
+    {Clauses, []} = insert_expr(Clauses0, Comments0),
     {{'receive', Meta, Clauses}, []};
-insert_nested({'receive', Meta, Clauses0, AfterExpr0, AfterBody0}, Comments0) ->
-    {Clauses, Comments1} = insert_expr_list(Clauses0, Comments0),
-    {AfterExpr, Comments2} = insert_expr(AfterExpr0, Comments1),
-    AfterBody = insert_expr_container(AfterBody0, Comments2),
-    {{'receive', Meta, Clauses, AfterExpr, AfterBody}, []};
+insert_nested({'receive', Meta, empty, {after_clause, _, _, _} = After0}, Comments0) ->
```

Comment:
```suggestion
insert_nested({'receive', Meta, empty, After0}, Comments0) ->
```
This can only be an `after_clause` node. The match is confusing compared to the next clause